### DISCUSSION TITLE
Error - attempt to concatenate field 'label' (a table value)

### DISCUSSION THREAD

**Discussion by madflow** [folke/noice.nvim#434] [Q&A] [answered]
I am getting an error when editing PHP files in a "Symfony" Project, and I cannot determine what is causing this, and if this plugin is to blame.

When I start typing a builtin PHP function and autocomplete it - I get the following error:

* NVIM v0.9.0
* LazyVim

```
2023-04-18T07:53:40 Messages  INFO "/private/tmp/my_project_directory/src/Kernel.php" 15L, 238B
2023-04-18T07:54:29 noice.nvim  ERROR ...l/share/nvim/lazy/noice.nvim/lua/noice/lsp/signature.lua:222: attempt to concatenate field 'label' (a table value)
2023-04-18T07:54:29 Messages  INFO table: 0x011aa1d6f8
```

![Kapture 2023-04-18 at 07 54 34](https://user-images.githubusercontent.com/183248/232684751-6ca564bc-5762-46c6-87a6-7c411a9730c5.gif)

Is there a way I can debug this and find out if this plugin is to blame?

I am currently only seeing this with PHP files. A minimal example would be:

```
<?php

namespace App;

class Kernel
{
    public function a(): void
    {
        \strstr()
    }
}
```
I have to add some kind of composer.json depenencies to make this appear.

**Comment by folke** [comment, marked as answer]
Should be fixed now. Would be great if you could confirm. Thanks!

**Comment by madflow** [comment]
@folke Thank you, works again!
